test/mpi/threads/perf/testlist

```diff
@@ -1 +1,2 @@
 mt_pt2pt_msgrate 2 arg=-nthreads=4
+mt_pt2pt_msgrate 2 arg=-nthreads=4 arg=-sync=lockall
```

Comment:
A typo: should be `mt_rma_msgrate`.

Shouldn't we add another extra to test `arg=-sync=fence`?